conversion/conversion.go

```diff
@@ -347,10 +350,22 @@ func dataFromDatabase(ctx context.Context, sourceProfile profiles.SourceProfile,
 				dashboardName = strings.Split(respDash.Name, "/")[3]
 				fmt.Printf("Monitoring Dashboard: %+v\n", dashboardName)
 			}
-
+			// store the generated resources locally in conv, this is used as source of truth for persistence and the UI (should change to persisted values)
 			streaming.StoreGeneratedResources(conv, streamingCfg, dfJobId, gcloudCmd, targetProfile.Conn.Sp.Project, "", internal.GcsResources{BucketName: gcsBucket}, dashboardName)
+			//persist job and shard level data in the metadata db
+			err = streaming.PersistJobExecutionData(ctx, targetProfile, sourceProfile, conv, migrationJobId, false)
+			if err != nil {
+				logger.Log.Info(fmt.Sprintf("Error storing generated resources in SMT metadata store...the migration job will still continue as intended. %v", err))
```

Comment:
Will it be better to modify the log msg to say job level data and shard level data respectively?

Reply:
Done